### PR TITLE
Bolt standardisation, brakes, battery cable into assembly tutorial

### PR DESCRIPTION
**Pull request by JanezCim**

### CODE REVIEW

**Review by dorkamotorka — APPROVED** [review]

**Comment by JanezCim** [comment]
Im going to merge this once the standardisation towers are in production, because people might rn still need the old instructions for the old towers

File: ConveyorBot/shell_tower/shell_tower_assembly.md [hunk 1]
```diff
@@ -70,7 +67,6 @@ What tools will be needed to assemble the Breadcrumb together:
  - small screwdriver (for M2 camera bolts)
  - big screwdriver (for mounting emergency switch)
  - M3 Allen key
- - M5 Allen key
  - M6 Allen key 
  - Paper tape
```
Comment (AljazKrsnik) [inline ConveyorBot/shell_tower/shell_tower_assembly.md:71]: TEST REVIEW

**Review by AljazKrsnik — COMMENTED** [review]
ALL GUT

File: ConveyorBot/shell_tower/shell_tower_assembly.md (hunk 1 above)
Comment (JanezCim) [reply ConveyorBot/shell_tower/shell_tower_assembly.md:71]: Yes, but. sgfasdfasd.
Sure ill fix this

**Comment by JanezCim** [comment]
@AljazKrsnik can you take another look?

**Comment by JanezCim** [comment]
Since we are porting all the tutorials to learn, I've already gone ahead and mergerd this branch into there: https://github.com/UbiquityRobotics/learn/pull/55/commits/f2d77d0ce691b5ed2717a1c8f9488eca44e2dc78. So this makes this PR obsolete since ALL of the breadcrumb documentation is going to get moved there.

**Pull request closed without merging**